test/unit-tests/cpptools.test.ts

```diff
@@ -15,15 +16,71 @@ function getTestResourceFilePath(filename: string): string {
 
 suite('CppTools tests', () => {
   test('Parse some compiler flags', () => {
-    let info = parseCompileFlags(['-DFOO=BAR']);
+    // Parse definition
+    const cpptoolsVersion = Version.v4;
```

Comment:
Maybe add one test for v3 that verifies we don't accidentally forget to use the version at some point?